projects/bp-gallery/src/components/views/archives/ArchiveView.tsx

```diff
@@ -142,6 +145,19 @@ const ArchiveView = ({ archiveId }: ArchiveViewProps) => {
               allowClicks={false}
               highQuality={true}
             />
+            {archive.email && (
+              <Button
+                fullWidth
+                className='!mt-2'
+                variant='contained'
+                endIcon={<ContactMail />}
+                onClick={() => {
+                  visit('/contact', { state: { archiveId: archive.id } });
+                }}
+              >
+                {t('archives.view.contact')}
+              </Button>
+            )}
```

Comment:
The contact button only shows when the archive has a showcase picture. Maybe adjust the conditional rendering so the two features render independently of one another? E. g. replace the `showcasePicture &&` with `(showcasePicture || archive.email) &&` and wrap the `PicturePreview` with a `{showcasePicture && ...}`.

Reply:
Oops, done!